htdocs/inbox/compose.bml

```diff
@@ -203,7 +203,7 @@ body<=
         $reply_u = $msg->other_u;
         $reply_to = $reply_u->display_name;
         $disabled_to = 1;
-        $msg_subject = $msg->subject || "(no subject)";
+        $msg_subject = LJ::dhtml( $msg->subject ) || "(no subject)";
```

Comment:
Would that open us up to an injection attack, or is subject_raw already cleaned at that point?